lib/src/Base/Graph/openturns/GraphImplementation.hxx

```diff
@@ -33,6 +33,7 @@ BEGIN_NAMESPACE_OPENTURNS
  * GraphImplementation implements graphic devices for plotting through R,
  * and manages drawables to be plotted on the same window
  */
+class Graph;
```

Comment:
Why do you need this?

Reply:
it was needed when gridlayout inerited from graphimplementation I think